CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to
 - Table of contents to documentation.
 - Get random element from array: `sample(array)`.
 - ECMAScript Modules support via importing '@metarhia/common/module'.
+- `Iterator#min()`, `Iterator#max()` and `Iterator#findCompare()` to
+  allow simpler iterator citerator#minonsumption in common use-cases.
```

Comment:
I don't understand the meaning of "to allow simpler iterator citerator#minonsumtion", can you elaborate?